### ISSUE TITLE
Hyperparameters for training

### ISSUE THREAD

**Issue by YoussefFathi** [vincentfung13/MINE#7]
Hello, 
would like to congratulate you on such great work!

Are the hyperparameters for the kitti_raw dataset included in params_kitti_raw.yaml the same ones used to reach the results in the paper or should they be changed?

**Comment by vincentfung13** [comment]
Hi Youssef, 

Yes you should be able to reproduce the performance by using params_kitti_raw.yaml, you can change it accordingly if you want to reproduce the results for different settings in the paper (for example the training resolution).

Zijian

**Comment by YoussefFathi** [comment]
I have tried recreating the data loader of KITTI Raw following the [code](https://github.com/google/layered-scene-inference/blob/59b5d37022f6aaab30dfd4ddcf560923eaf38578/lsi/data/kitti/data.py#L161) you specified in a previous issue.  I also deactivated the sparse disparity loss as mentioned in the paper and used the same hyperparams in params_kitti_raw.config. However, the model is not converging after training for 1 Day on 4 V100 GPUs. Is it possible to share a code snippet of the KITTI Raw Data loader to verify that I implemented it correctly?

**Comment by Stephanie-ustc** [comment]
Look forward to friends who pay attention to this work and hope to discuss it. Thank you.
WeChat link as follows：
https://wx3.sinaimg.cn/mw690/b0e12477gy1gwcfjj9ez9j20tc10xjuv.jpg @YoussefFathi

**Comment by YoussefFathi** [comment]
It was mentioned in the MPI paper that for KITTI Raw they cropped all images by 5% from all sides before testing and also calculated the metrics only on non-occluded regions of the scene. Did you do the last part in your method or only cropped by 5% from all sides?

**Comment by hyBlue** [comment]
@YoussefFathi 
Hi, did you figure it out how to evaluate? I'm now testing the author's pretrained model but I can never get the results as in the paper. I evaluated PSNR and it seems around 15. on in the test set. Could you please share the tips if you get the good results?
